frontend/lib/src/components/core/Block/styled-components.ts

```diff
@@ -113,10 +113,11 @@ export const StyledElementContainer = styled.div<StyledElementContainerProps>(
 interface StyledColumnProps {
   weight: number
   gap: string
+  verticalAlignment?: "top" | "center" | "bottom"
```

Comment:
nit: Should probably centralize the type for vertical alignment.

Reply:
Instead of creating a new type for this, I changed it to reuse the existing enum type from our protos. Initially, I didn't want to mix proto & styled-components, but I don't think there is a reasonable argument for why not to do that